app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,17 @@
+module API
+  module V1
+    class BookSuggestionsController < ApiController
+      def create
+        book_suggestion = BookSuggestion.create!(book_suggestion_params)
+        render json: book_suggestion, status: :created
```

Comment:
this can be joined in the same line, like 
`render json: BookSuggestion.create!(book_suggestion_params), status: :created`, just as a comment 😛

Reply:
😃 right! done!